#166007544 Build Middleware To Check For Whitespace in Passwords

elikeyz opened the pull request:
#### What does this PR do?
Add an additional middleware to the signup endpoint that screens out passwords that contain whitespaces
#### Description of Task to be completed?
- Create the middleware to check the passwords for whitespaces
- Write tests for this middleware
- Remove password from user endpoints response body
#### How should this be manually tested?
Clone this repo, run ```npm install``` to install the dependencies and run ```npm start``` or ```npm run start-dev``` to start the application and then navigate to ```localhost:3000``` on Postman.
#### What are the relevant pivotal tracker stories?
#166007544

elikeyz committed:
feature(users): remove password from user routes responses

remove the password from the response object of user routes

[Finishes #166007544]

elikeyz committed:
feature(signup): create a middleware to check for whitespaces in password

- create a middleware to check that sends an error response if the signup
  password inputted by the user has whitespaces in it
- write a test for this middleware

[Finishes #166007544]

Danielshow commented on src/utils/routes.js:
```diff
@@ -31,7 +32,7 @@ const {
 const { getLoanRepayments, postClientRepaymentTranx } = repaymentsController;
 
-router.post('/auth/signup', loginValidate, signupValidate, signupValidate2, signupVerify, signup);
+router.post('/auth/signup', loginValidate, signupValidate, signupValidate2, signupValidate3, signupVerify, signup);
```
Why this? You can use good naming conventions here

Danielshow reviewed (commented):
Attend to this

elikeyz committed:
refactor(middleware): rename middleware files and functions

- rename middleware files and functions to adhere to good naming
  conventions

[Finishes #166007544]
